modules/update-upgrading-cli.adoc

```diff
@@ -59,7 +55,7 @@ For details and information on how to perform an `EUS-to-EUS` channel update, pl
 _Preparing to perform an EUS-to-EUS upgrade_ page, listed in the Additional resources section.
 ====
 
-. Based on your organization requirements, set the appropriate update channel. For example, you can set your channel to `stable-4.12`, `fast-4.12`, or `eus-4.12`. For more information about channels, refer to _Understanding update channels and releases_ listed in the Additional resources section.
+. Based on your organization requirements, set the appropriate update channel. For example, you can set your channel to `stable-4.13`, `fast-4.13`, or `eus-4.13`. For more information about channels, refer to _Understanding update channels and releases_ listed in the Additional resources section.
```

Comment:
There's no `eus-4.13`, it's only for even releases.  So 4.13.z releases will only show up in an `eus-4.14` channel, and even that won't happen until [45 to 90 days after 4.14 GAs][1].

[1]: https://docs.openshift.com/container-platform/4.14/updating/understanding_updates/understanding-update-channels-release.html#fast-stable-channel-strategies_understanding-update-channels-releases